RecoLocalCalo/HcalRecProducers/src/HcalSimpleReconstructor.cc

```diff
@@ -157,3 +128,34 @@ void HcalSimpleReconstructor::produce(edm::Event& e, const edm::EventSetup& even
     }
   } 
 }
+
+void
+HcalSimpleReconstructor::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+
+
+  // horeco
+  edm::ParameterSetDescription descHO;
+  descHO.add<double>("correctionPhaseNS", 13.0);
+  descHO.add<edm::InputTag>("digiLabel", edm::InputTag("hcalDigis"));
+  descHO.add<bool>("tsFromDB", true);
+  descHO.add<int>("samplesToAdd", 4);
+  descHO.add<std::string>("Subdetector", "HO");
+  descHO.add<bool>("correctForTimeslew", true);
+  descHO.add<bool>("dropZSmarkedPassed", true);
+  descHO.add<bool>("correctForPhaseContainment", true);
+  descHO.add<int>("firstSample", 4);
+  descriptions.add("hosimplereco", descHO);
```

Comment:
NB: this will lead to hosimplereco_cfi.py and hfsimplereco_cfi.py files created in 
$CMSSW_BASE/cfipython/$SCRAM_ARCH/RecoLocalCalo/HcalRecProducers/
which are visible for parsing.
So, you can use them as e.g.  RecoLocalCalo.HcalRecProducers.hosimplereco_cfi.hosimplereco everywhere an instance of the module is needed